orb-ui/cone/src/led.rs

```diff
@@ -0,0 +1,248 @@
+use crate::CONE_FTDI_LED_INDEX;
+use color_eyre::eyre;
+use color_eyre::eyre::{eyre, Context};
+use ftdi_embedded_hal::eh1::spi::SpiBus;
+use ftdi_embedded_hal::libftd2xx::{Ft4232h, Ftdi, FtdiCommon};
+use orb_rgb::Argb;
+use tokio::sync::{mpsc, oneshot};
+use tokio::task;
+
+pub const CONE_LED_COUNT: usize = 64;
+
+/// LED strip handle.
+/// To send new values to the LED strip.
+pub struct LedStrip {
+    /// Used to signal that the task should be cleanly terminated.
+    pub kill_tx: oneshot::Sender<()>,
+    tx: mpsc::Sender<[Argb; CONE_LED_COUNT]>,
+}
+
+pub struct LedJoinHandle(pub task::JoinHandle<eyre::Result<()>>);
```

Comment:
Always #[derive(Debug)]. Should probably turn on that lint in clippy.

Reply:
why? 🤔 
should I do it on `LedStrip` above as well?